zio-schema/shared/src/test/scala/zio/schema/SchemaGen.scala

```diff
@@ -301,13 +311,176 @@ object SchemaGen {
     )
 
   lazy val anySchema: Gen[Random with Sized, Schema[_]] = Gen.oneOf(
-    anyPrimitive,
-    anyOptional,
-    anyTuple,
-    anySequence,
-    anyEnumeration,
-    anyRecord,
-    anyTransform,
-    anyEither
+    // anyPrimitive,
+    anyOptional(anyPrimitive)
```

Comment:
Dead code or intentional?

Reply:
Unintentional. Commented while debugging and forgot to uncomment :( Fixed